examples/kubernetes/hpe-3par-primera/secret.yaml

```diff
@@ -0,0 +1,13 @@
+apiVersion: v1
+kind: Secret
+metadata:
+  name: hpe-secret
```

Comment:
Please change this to `primera3par-secret` and have that reflected in the remaining examples.

Reply:
@drajen Done.